context/AuthContextProvider.tsx

```diff
@@ -40,6 +41,7 @@ export const AuthContextProvider = ({
   children: React.ReactNode;
 }): JSX.Element => {
   const [isSignedIn, setIsSignedIn] = useState<boolean>(false);
+  const [isSuperAdmin, setIsSuperAdmin] = useState<boolean>(false);
```

Comment:
I don't believe this needs to be a useState. It's going to cause a rehydration of the entire application that may be alreayd handled from other auth functions.

Reply:
✅ done.